bazel/external/requirements.txt

```diff
@@ -1632,3 +1632,6 @@ setuptools==49.6.0 \
     #   -r requirements.in
     #   grpcio-tools
     #   jsonschema
+
+sdnotify==0.3.2 \
+    --hash=sha256:73977fc746b36cc41184dd43c3fe81323e7b8b06c2bb0826c4f59a20c56bb9f1
```

Comment:
Maybe we could add a 
`    # via -r requirements.in`
in the line below. This should have been auto-generated.